server/protocols/webadmin/cli/james_cli

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+
+import click
+import json
+import requests as req
+import sys
+
+
+def print_json(obj):
+    parsed = json.loads(obj.text)
+    print(json.dumps(parsed, indent=4, sort_keys=True))
+
+
+@click.group()
+@click.option("--ip", default="127.0.0.1", show_default=True, help="Set ip address")
```

Comment:
Should be `--host` IMO

Reply:
why not an URL instead? It would allow to handle http / https.